.gitignore

```diff
@@ -65,6 +65,9 @@ python-env/
 .idea
 
 Carthage/
+# Required for Glean
+/Generated
```

Comment:
`/Client/Generated` I believe (when I tried the patch locally I saw it wasn't ignoring the file so I think that is the problem)

Reply:
Easy fix - will do!